python/lsst/ts/standardscripts/enable_group.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["EnableGroup"]
+
+import abc
+
+from lsst.ts import salobj
+
+
+class EnableGroup(salobj.BaseScript, metaclass=abc.ABCMeta):
+    """Base Script for enabling groups of CSCs.
+
+    This base class is setup to operate with minimum configuration. By default
+    it will try to access an attribute `self.config.ignore`, which is supposed
+    to contain a list of CSCs from the group to be ignored in the process.
+    Nevertheless, if the parent class does not provide an `ignore` property it
+    will be ignored.
+
+    In addition, when subclassing this base class, users can provide settings
+    for the configurable CSCs in the group. The name of the CSC in the
+    configuration must match the name of the CSC in `group.components`, which
+    is the name of the CSC in lowercase, replacing the ":" with "_" for indexed
+    components. For example,
+
+        * ATMCS -> atmcs
+        * NewMTMount -> newmtmount
+        * Hexapod:1 -> hexapod_1
```

Comment:
An example of "settings for the configurable CSCs" would be a really big help here.

I think it's just a name with an associated string that is the value for settingsToApply (which I know is going to be renamed). But I'm not sure.